frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -220,9 +277,13 @@ export const Send = () => {
     setStep('sending');
     setSendError(undefined);
     try {
-      switch (parsedInput?.type) {
+      switch (paymentDetails?.type) {
       case InputTypeVariant.BOLT11:
-        await postSendPayment({ bolt11: parsedInput.invoice.bolt11 });
+        await postSendPayment({
+          bolt11: paymentDetails.invoice.bolt11,
+          // amountMsat is optional, if amount is missing the UI shows edit-invoice step for the user to enter a custom amountMsat which is passed here
+          amountMsat: customAmount
```

Comment:
AFAIK amountMsat unit is milli satoshi. The amount should be multiplied * 1000

Reply:
oups 😇